impl/src/test/java/org/ehcache/config/builders/CacheManagerBuilderTest.java

```diff
@@ -76,31 +79,34 @@ public void testCanOverrideSerializerConfig() {
     assertThat(managerBuilder.withSerializer(String.class, serializer2)).isNotNull();
   }
 
-  @Test(expected = IllegalArgumentException.class)
-  public void testDuplicateServiceCreationConfigurationFails() {
-    newCacheManagerBuilder().using(new DefaultCopyProviderConfiguration())
-        .using(new DefaultCopyProviderConfiguration());
+  @Test
+  public void testDuplicateServiceCreationConfigurationOk() {
+    assertThat(newCacheManagerBuilder().using(new DefaultCopyProviderConfiguration())
```

Comment:
Do we have tests that assure like configurations aren't duplicated?

Reply:
You mean that 'incompatible' configurations that can't tolerate multiple instances are correctly removed? I just added coverage to the configuration builder for that. We could add it at this layer too.